frontend/src/components/dropdown/DropDownComponent.jsx

```diff
@@ -0,0 +1,54 @@
+import React, { Component } from 'react';
+import theme from 'resources/theme';
+
+/**
+ * a seperate DropDownComponent
+ * here will be decided if the number of variables are more than one
+ * then a drop down list should be shown
+ * otherwise there is nothing to select
+ *
+ * @author Parham Gandomkar
+ */
```

Comment:
👍 

Would be cool though, if the case where there is only one model would also be displayed in the same style. The style where there are multiple models.

Reply:
I like how you did that!